src/trunk.c

```diff
@@ -1492,7 +1502,9 @@ trunk_add_pivot_new_root(trunk_handle *spl,
 {
    const char *pivot_key                       = trunk_get_pivot(spl, child, 0);
    __attribute__((unused)) const char *min_key = spl->cfg.data_cfg->min_key;
-   debug_assert(trunk_key_compare(spl, pivot_key, min_key) == 0);
+   __attribute__((unused)) const int   key_cmp_rv =
+      trunk_key_compare(spl, pivot_key, min_key);
+   debug_assert((key_cmp_rv == 0), "key_cmp_rv=%d\n", key_cmp_rv);
```

Comment:
Change will be retained as it is useful on its own.

Reply:
Use the `debug_only` macro here for `key_cmp_rv`

Reply:
Done. Nice tip, @ajhconway -- Forgot that this existed.

What's more .. the line now fits within 80-chars, so no more annoying `clang-format` re-indents on this line:

```
+   debug_only const int key_cmp_rv = trunk_key_compare(spl, pivot_key, min_key);
```